test/functional/viml/msgpack_functions_spec.lua

```diff
@@ -0,0 +1,606 @@
+local helpers = require('test.functional.helpers')
```

Comment:
FYI there is now a `test/functional/eval/` directory, I suggest putting this there. I will move the other resident of this `viml/` directory somewhere else.